packages/components/src/components/LoadingBar/LoadingBar.tsx

```diff
@@ -55,7 +55,13 @@ export const LoadingBar = forwardRef<ElementRef<typeof ProgressRadixPrimitive.Ro
                 aria-busy={value !== max}
                 value={value}
                 max={max}
-                style={value ? ({ '--loading-bar-value': `${value}%` } as CSSProperties) : {}}
+                style={
+                    {
+                        '--loading-bar-value': value,
+                        '--loading-bar-max': max,
+                        '--loading-bar-proportion': 'calc(var(--loading-bar-value) / var(--loading-bar-max))',
```

Comment:
could that be in the styles file instead of being a separate css variable

Reply:
Could be easily in the TW arbitrary class, but I use it to test it is the correct value we are passing in, else we don't have a way to test it "for real"

Reply:
If it bother you, we can make another PR to inline it and see how we can change the tests